tests/ctst/tsconfig.json

```diff
@@ -26,6 +26,7 @@
       // "strictPropertyInitialization": true,  /* Enable strict checking of property initialization in classes. */
       "noImplicitThis": true,                   /* Raise error on 'this' expressions with an implied 'any' type. */
       // "alwaysStrict": true,                  /* Parse in strict mode and emit "use strict" for each source file. */
+      "skipLibCheck": true,
```

Comment:
do we really want to save compilation time at the expense of type-system accuracy?

Reply:
It is because of issues with some libs we do not control, e.g.:

```
node_modules/cli-testing/node_modules/@aws-sdk/client-s3/dist-types/auth/httpAuthSchemeProvider.d.ts:1:10 - error TS2724: '"@aws-sdk/core"' has no exported member named 'AwsSdkSigV4AAuthInputConfig'. Did you mean 'AwsSdkSigV4AuthInputConfig'?
```

And one vaultclient issue (ticket is VLTCLT-52):
```
node_modules/vaultclient/lib/IAMClient.d.ts:266:100 - error TS1016: A required parameter cannot follow an optional parameter.

266     verifySignatureV4(stringToSign: string, signature: string, accessKey: string, region?: string, scopeDate: string, options: {
```

I will try to remove this line when the sdk is more up to date.